doc: Provide valid alternative for import in `cloud_provider_access_authorization` (#3101)
mongodb/terraform-provider-mongodbatlas: docs/resources/cloud_provider_access.md

## Import mongodbatlas_cloud_provider_access_authorization

> The Cloud Provider Access Authorization resource cannot be imported into Terraform. If this resource needs to be imported, you can safely re-create the resource. If the associated role has already been authorized, issuing a new authorization request will simply succeed without any adverse effects.

Reviewer: [q] regarding "In this case it would mean ensuring that the authorization PATCH call was made for the provided role." I'm curious what happens if the PATCH is never made for this role? What does the GET return in that case? can that response be used to identify the role was never authorized?

Author: The [getProviderAccessRole](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Cloud-Provider-Access/operation/getCloudProviderAccessRole) method returns the existing role. For the case of AWS `authorizedDate` can be used, for the case of Azure `createdDate` and `lastUpdatedDate` could be used but this might be more risky as it is not explicitly related to the authorization.

Reviewer 2: ```suggestion
You cannot import the Cloud Provider Access Authorization resource into Terraform. Instead, after authorizing the associated role, you can recreate the resource.
```

Author: incorporated feedback with a small adjustment, let me know if this looks good
